storage/container_storage_interface/persistent-storage-csi-azure-file.adoc

```diff
@@ -0,0 +1,40 @@
+[id="persistent-storage-csi-azure-file"]
+= Azure File CSI Driver Operator
+include::modules/common-attributes.adoc[]
+:context: persistent-storage-csi-azure-file
+
+toc::[]
+
+== Overview
+
+{product-title} is capable of provisioning persistent volumes (PVs) by using the Container Storage Interface (CSI) driver for Microsoft Azure File Storage.
+
+:FeatureName: Azure File CSI Driver Operator
+include::modules/technology-preview.adoc[leveloffset=+1]
+
+Familiarity with xref:../../storage/understanding-persistent-storage.adoc#understanding-persistent-storage[persistent storage] and xref:../../storage/container_storage_interface/persistent-storage-csi.adoc#persistent-storage-csi[configuring CSI volumes] is recommended when working with a CSI Operator and driver.
+
+To create CSI-provisioned PVs that mount to Azure File storage assets with this feature enabled, {product-title} installs the Azure File CSI Driver Operator and the Azure File CSI driver by default in the `openshift-cluster-csi-drivers` namespace.
+
+* The _Azure File CSI Driver Operator_, after being enabled, provides a storage class that is named `azurefile-csi` that you can use to create persistent volume claims (PVCs). The Azure File CSI Driver Operator supports dynamic volume provisioning by allowing storage volumes to be created on-demand, eliminating the need for cluster administrators to pre-provision storage.
+
+* The _Azure File CSI driver_ enables you to create and mount Azure File PVs.
+
+include::modules/persistent-storage-csi-about.adoc[leveloffset=+1]
+
+[IMPORTANT]
+====
+{product-title} defaults to using an in-tree, non-CSI, plug-in to provision Azure File storage.
```

Comment:
I'm not sure if this is correct. By default, OCP doesn't create any Azure File StorageClass (not even an in-tree one).

Reply:
Yes, when installing an OCP cluster, there is no Azure file in-tree StorageClass installed (Although we have documentation to install it), maybe only mention we support  Azure file in-tree here if we need to talk about CSI migration?